ArcGISRuntimeSDKQt_QMLSamples/EditData/AddFeaturesFeatureService/README.md

```diff
@@ -1,20 +1,35 @@
-# Add features (feature service)
+# Add features
```

Comment:
Let's leave the names as-is. We will have a follow on task to clean up names next release